core/src/edu/cornell/gdiac/rabbeat/GameController.java

```diff
@@ -157,6 +157,10 @@ public class GameController implements Screen, ContactListener {
 
 	private int SFXVolume = 10;
 
+	private Enemy[] boundedEnemies;
+	
+	private BoxGameObject[] boundedPlatforms;
```

Comment:
this should be in objectcontroller

Reply:
you can revert the changes, they're not being used yet. They're supposed to represent the list of enemies that are bounded to moving/weighted platforms